setup.py

```diff
@@ -18,21 +18,21 @@
     "xgi.utils",
 ]
 
-version = "0.1"
+version = "0.1.1"
 
 authors = "Nicholas Landry and Leo Torres"
 
 author_email = "nicholas.landry@colorado.edu"
 
 url = "https://github.com/ComplexGroupInteractions/xgi"
 
-description = "XGI is a Python library for the representation and analysis of complex systems with group (higher-order) interaction."
+description = "XGI is a Python library for the representation and analysis of complex systems with group (higher-order) interactions."
 
 install_requires = [
     "networkx>=2.2,<3.0",
-    "numpy>=1.15.0,<2.0",
-    "scipy>=1.1.0,<2.0",
-    "pandas>=0.23",
+    "numpy>=1.19.0,<1.22.0",
+    "scipy>=1.5.0,<1.7.0",
+    "pandas>=0.23.4,<1.0.0",
```

Comment:
Comment (non-blocking): in the future, it'd be best if the `setup.py` file just reads the contents of `requirements/default.txt` so we don't have to remember to keep them in sync.

Reply:
I implemented this because it was an easy modification